plugin/common/common.cuh

```diff
@@ -35,7 +35,6 @@ __device__ inline T rsqrt(const T& x);
 template <typename T>
 __device__ inline T exp(const T& x);
 
-
```

Comment:
could you also revert the changes of blank lines, this could help us to do the integration between public repo and internal repo, thanks!

Reply:
No problem!  I have reverted them. Looking forward to your review.